tests/filecheck/dialects/func/func_invalid.mlir

```diff
@@ -31,3 +31,9 @@ builtin.module {
 }
 
 // CHECK: Expected all arguments to be named or all arguments to be unnamed.
+
+// -----
+
+"func.call"() { "callee" = @call::@invalid } : () -> ()
+
+// CHECK:  Operation does not verify: expected empty array, but got ["invalid"]
```

Comment:
This does not feel like an actionable error message. What is an array in this context? Can we somehow make this understandable?

Reply:
We have an idea, but that's going to be for another PR.
Essentially we'll add a new constraint that will catch an error, and return it.
For now at least, this error is better than the previous one ;)

Reply:
#1579